### ISSUE TITLE
Bulk copy WITH HEADER=true not working for user defined types

### ISSUE THREAD

**Issue by mmatczuk** [scylladb/scylla#2446]
*Installation details*
Scylla version (or git commit hash): c8cb3d6
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS
Platform (physical/VM/cloud instance type/docker): docker

Copy seems not to use CSV headers when importing data to user defined type cell i.e.

I have a table `videos` with encoding of type `video_encoding`

```cqlsh:foobar> DESCRIBE videos

CREATE TABLE foobar.videos (
    video_id timeuuid PRIMARY KEY,
    encoding frozen<video_encoding>,
) WITH bloom_filter_fp_chance = 0.01
    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
    AND comment = ''
    AND compaction = {'class': 'SizeTieredCompactionStrategy'}
    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
    AND crc_check_chance = 1.0
    AND dclocal_read_repair_chance = 0.1
    AND default_time_to_live = 0
    AND gc_grace_seconds = 864000
    AND max_index_interval = 2048
    AND memtable_flush_period_in_ms = 0
    AND min_index_interval = 128
    AND read_repair_chance = 0.0
    AND speculative_retry = '99.0PERCENTILE';

cqlsh:foobar> DESCRIBE TYPE video_encoding 

CREATE TYPE foobar.video_encoding (
    bit_rates frozen<set<text>>,
    encoding text,
    height int,
    width int
);
```

I have a CSV file that looks like this (note different fields / cells order):

```
video id,video encoding
2644c36e-14bd-11e5-839e-8438355b7e3a,"{encoding: '1080p', height: 1080, width: 1920, bit_rates: {'3000 Kbps', '4500 Kbps', '6000 Kbps'}}"
2645e79c-14bd-11e5-a456-8438355b7e3a,"{encoding: '1080p', height: 1080, width: 1920, bit_rates: {'3000 Kbps', '4500 Kbps', '6000 Kbps'}}"
2645e8d1-14bd-11e5-958e-8438355b7e3a,"{encoding: '1080p', height: 1080, width: 1920, bit_rates: {'3000 Kbps', '4500 Kbps', '6000 Kbps'}}"
2645e9a3-14bd-11e5-9bc5-8438355b7e3a,"{encoding: '1080p', height: 1080, width: 1920, bit_rates: {'3000 Kbps', '4500 Kbps', '6000 Kbps'}}"
```
`COPY` command fails trying to parse `set<text>` as `int`

```
cqlsh:foobar> COPY videos (video_id, encoding) FROM 'videos_encoding.csv' WITH HEADER=true;
Reading options from the command line: {'header': 'true'}
Using 3 child processes

Starting copy of foobar.videos with columns [video_id, encoding].
Failed to import 430 rows: ParseError - Failed to parse {encoding: '1080p', height: 1080, width: 1920, bit_rates: {'3000 Kbps', '4500 Kbps', '6000 Kbps'}} : invalid literal for int() with base 10: "{'3000 Kbps', '4500 Kbps', '6000 Kbps'}",  given up without retries
Failed to process 430 rows; failed rows written to import_foobar_videos.err
```

**Comment by duarten** [comment]
This seems to be a problem with `cqlsh`. Did this work with Cassandra? Which version?

**Comment by duarten** [comment]
Seems like a duplicate of #2393.

**Comment by duarten** [comment]
Seems fixed by apache/cassandra@bdca25e96e8fc9b6379c599ef001a256e474688e

**Comment by mmatczuk** [comment]
Yep, I'm closing the issue.
